Description: Upstream changes introduced in version 0.120-r1-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 q4wine (0.120-r1-2) unstable; urgency=low
 .
   * Edited debian/rules to fix libq4wine-core.so destination
     from /usr/lib64/q4wine/ to /usr/lib/q4wine/ (Closes: #612027)
   * Added debian/patches/
     - possible fix for Debian GNU/kFreeBSD
     - possible fix for Debian GNU/Hurd
 .
 The person named in the Author field signed this changelog entry.
Author: Boris Pek <tehnick-8@mail.ru>
Bug-Debian: http://bugs.debian.org/612027

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- q4wine-0.120-r1.orig/src/q4wine-gui/wizard.cpp
+++ q4wine-0.120-r1/src/q4wine-gui/wizard.cpp
@@ -174,6 +174,11 @@ Wizard::Wizard(int WizardType, QString v
 
     updateScena();
 
+#ifdef _OS_FREEBSD_
+    this->comboMountProfiles->removeItem(3);
+    this->comboMountProfiles->removeItem(2);
+#endif
+
     this->installEventFilter(this);
 
     return;
--- q4wine-0.120-r1.orig/src/q4wine-gui/appsettings.cpp
+++ q4wine-0.120-r1/src/q4wine-gui/appsettings.cpp
@@ -222,8 +222,8 @@ AppSettings::AppSettings(QWidget * paren
     settings.endGroup();
 
 #ifdef _OS_FREEBSD_
-    radioFuse->setEnabled(FALSE);
-    radioEmbedded->setEnabled(FALSE);
+    this->comboMountProfiles->removeItem(3);
+    this->comboMountProfiles->removeItem(2);
 #endif
 
     settings.beginGroup("advanced");
--- q4wine-0.120-r1.orig/src/q4wine-lib/q4wine-lib.cpp
+++ q4wine-0.120-r1/src/q4wine-lib/q4wine-lib.cpp
@@ -604,9 +604,10 @@ QStringList corelib::getCdromDevices(voi
 
 #ifdef _OS_LINUX_
         if (fileInfo.fileName().contains(QRegExp("^cdrom")) or fileInfo.fileName().contains(QRegExp("^sr")) or fileInfo.fileName().contains(QRegExp("^dvd"))){
-#endif
-#ifdef _OS_FREEBSD_
+#elif _OS_FREEBSD_
             if (fileInfo.fileName().contains(QRegExp("^cdrom")) or fileInfo.fileName().contains(QRegExp("^cd")) or fileInfo.fileName().contains(QRegExp("^acd")) or fileInfo.fileName().contains(QRegExp("^dvd"))){
+#else
+            if (fileInfo.fileName().contains(QRegExp("^cdrom")) or fileInfo.fileName().contains(QRegExp("^sr")) or fileInfo.fileName().contains(QRegExp("^dvd"))){
 #endif
                 if (fileInfo.isSymLink()){
                     if (!retVal.contains(fileInfo.symLinkTarget()))
